From 8b18a44fa7e5ddf6c8caee37de4f6112c64dfc87 Mon Sep 17 00:00:00 2001
From: Sergei Golubchik <serg@mariadb.org>
Date: Mon, 13 Nov 2017 13:11:53 +0800
Subject: [PATCH] mroonga after-merge CMakeLists.txt fixes

1. remove erroneously committed *.orig
2. fix LZ4 detection on Mac OS X and FreeBSD. Cannot do

   pkg_check_modules(LIBLZ4 liblz4)
   find_library(LIBLZ4_LIBS ... )

because find_library(X) does not do anything if X is defined (documented),
and pkg_check_modules(Y) sets Y_LIBS to "" (undocumented!)
---
 storage/mroonga/vendor/groonga/CMakeLists.txt      |   4 +-
 storage/mroonga/vendor/groonga/lib/CMakeLists.txt  |   2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)
 delete mode 100644 storage/mroonga/CMakeLists.txt.orig
 delete mode 100644 storage/mroonga/vendor/groonga/CMakeLists.txt.orig

diff --git a/storage/mroonga/vendor/groonga/CMakeLists.txt b/storage/mroonga/vendor/groonga/CMakeLists.txt
index 3d957c9d5152..e27070f9e0c3 100644
--- a/storage/mroonga/vendor/groonga/CMakeLists.txt
+++ b/storage/mroonga/vendor/groonga/CMakeLists.txt
@@ -355,13 +355,13 @@ if(NOT ${GRN_WITH_LZ4} STREQUAL "no")
   if(GRN_WITH_BUNDLED_LZ4)
     set(LIBLZ4_INCLUDE_DIRS
       "${CMAKE_CURRENT_SOURCE_DIR}/vendor/lz4-${GRN_BUNDLED_LZ4_VERSION}/lib")
-    set(LIBLZ4_LIBS liblz4)
+    set(LZ4_LIBS liblz4)
   else()
     if(NOT DEFINED LIBLZ4_FOUND)
       pkg_check_modules(LIBLZ4 liblz4)
     endif()
     if(LIBLZ4_FOUND)
-      find_library(LIBLZ4_LIBS
+      find_library(LZ4_LIBS
 	NAMES ${LIBLZ4_LIBRARIES}
 	PATHS ${LIBLZ4_LIBRARY_DIRS}
 	NO_DEFAULT_PATH)
diff --git a/storage/mroonga/vendor/groonga/lib/CMakeLists.txt b/storage/mroonga/vendor/groonga/lib/CMakeLists.txt
index a90cea0043eb..6765261feb7d 100644
--- a/storage/mroonga/vendor/groonga/lib/CMakeLists.txt
+++ b/storage/mroonga/vendor/groonga/lib/CMakeLists.txt
@@ -94,7 +94,7 @@ set(GRN_ALL_LIBRARIES
     ${RT_LIBS}
     ${PTHREAD_LIBS}
     ${Z_LIBS}
-    ${LIBLZ4_LIBS}
+    ${LZ4_LIBS}
     ${LIBZSTD_LIBS}
     ${MESSAGE_PACK_LIBS}
     ${DL_LIBS}
